dashboard/config/libraries/variableLog.interpreted.js

```diff
@@ -1,78 +1,110 @@
 var studentVarToken = false;
-var varLog={};
-if(!validationProps){
-  var validationProps ={};
-}
+var previousVarLog;
 
-function updateLog(){
-  var start = Object.keys(window).indexOf("studentVarToken")+1;
-  var end = Object.keys(window).indexOf("__jsCB");
+// Create an array of objects representing within the variables that have been
+// declared or updated within the Blockly environment. All Blockly variables
+// are global, and we would like to find a more efficient way to find just
+// those used by the student.
+function buildVariableLog() {
+  text("Building variable log...",0,15);
```

Comment:
Who sees this text?

Reply:
Removed. This text was for me to help identify a now-fixed bug where this function was being called too often.